.travis.yml

```diff
@@ -38,4 +38,6 @@ script:
   - for toolkit in ${TOOLKITS}; do edm run -- python etstool.py test --runtime=${RUNTIME} --toolkit=${toolkit} || exit; done
 after_success:
   - edm run -- python etstool.py flake8
+  - ls
```

Comment:
Remove?

Reply:
Yes, removed in the most up to date version. Thanks!